06.bowling_object/game.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class Game
+  LAST_FRAME = 9
+
+  def initialize(marks)
+    @frames = create_frames(marks)
+  end
+
+  def score
+    @frames.each_with_index.sum do |frame, index|
+      next frame.score if index == LAST_FRAME
+
+      if frame.strike?
+        frame.score + next_two_shots_score(index)
+      elsif frame.spare?
+        frame.score + next_shot_score(index)
+      else
+        frame.score
+      end
```

Comment:
```suggestion
      frame.score(@frames)
```

こんな感じにしてFrameの計算はFrame自身が全部担当する、という形にするとクールかもしれません。
indexに関してはFrameをnewするときに数値を渡せるはずです。

Reply:
フレームクラスで追加得点の計算処理を行うようにしました。